package.json

```diff
@@ -69,6 +69,8 @@
   "dependencies": {
     "entities": "^2.1.0",
     "fast-xml-parser": "^3.17.4",
+    "lodash.debounce": "^4.0.8",
+    "lodash.throttle": "^4.1.1",
```

Comment:
:question: can you provide figures about the package size increase please?

Reply:
- For minified build

with lodash it is **153.59 KB**

with mocked functions **150.13 KB**
so ~3.5 KB

``` javascript
  debounce(callback: (event: Event, up: boolean) => void, delay: number) {
    return callback;
  }
  throttle(callback: (event: Event, up: boolean) => void, delay: number) {
    return callback;
  }
```

- For non minified:
app - 189.28 KB (65.32%)
fast-xml-parser - 42.41 KB (14.63%)
entities - 33.1 KB (11.42%)
lodash.throttle - 13 KB (4.49%)
lodash.debounce - 10.53 KB (3.63%)
rollup helpers - 1.47 KB (0.51%)

Reply:
see #823 alternative without lodash

154.59 KB (lodash)     vs     151.3 KB (custom)